patch.go

```diff
@@ -0,0 +1,59 @@
+package quicktest
```

Comment:
Add license header: `// Licensed under the MIT license, see LICENCE file for details.`

Reply:
done